akka-remote/src/main/scala/akka/remote/PhiAccrualFailureDetector.scala

```diff
@@ -58,7 +55,8 @@ class PhiAccrualFailureDetector(
   val maxSampleSize:            Int,
   val minStdDeviation:          FiniteDuration,
   val acceptableHeartbeatPause: FiniteDuration,
-  val firstHeartbeatEstimate:   FiniteDuration)(
+  val firstHeartbeatEstimate:   FiniteDuration,
+  eventStream:                  EventStream)(
   implicit
   clock: Clock) extends FailureDetector {
 
```

Comment:
for backwards compatibility you must also add a constructor (`def this(`) that corresponds to previous old constructor